msg/tools/generate_microRTPS_bridge.py

```diff
@@ -157,40 +156,108 @@ def check_rtps_id_uniqueness(classifier):
 
 parser = argparse.ArgumentParser()
 
-parser.add_argument("-a", "--agent", dest='agent', action="store_true",
-                    help="Flag for generate the agent, by default is true if -c is not specified")
-parser.add_argument("-c", "--client", dest='client', action="store_true",
-                    help="Flag for generate the client, by default is true if -a is not specified")
-parser.add_argument("-i", "--generate-idl", dest='gen_idl',
-                    action="store_true", help="Flag for generate idl files for each msg")
+parser.add_argument(
+    "-a",
```

Comment:
Can we have fewer newlines here?

Reply:
This is all `autopep8` doing. I mean I don't mind having the lines broken like this, but if you consider it is better for readability, I can always clean it.